src/proto/streams/prioritize.rs

```diff
@@ -613,6 +626,11 @@ impl Prioritize {
 
                     trace!("pop_frame; frame={:?}", frame);
 
+                    if cfg!(debug_assertions) && stream.state.is_idle() {
```

Comment:
If we're going to store a field on the struct, we might as well make it an assert (not debug).

Reply:
I wanted to make the field itself only part of the struct on debug, but since `if` statements can't have `#[cfg(...)]` attrs, the assertion needed to go in its own function, and at that point, I punted.